script/domVisitor.ts

```diff
@@ -86,7 +86,7 @@ async function visitPages(
         anyFailures = true;
         bar.interrupt(`- (${matches.length}) ${pageUrl}#${matches[0]}`);
       }
-    } catch (e) {
+    } catch (e: any) {
```

Comment:
in a perfect world all these errors would be typed as `unknown` and we'd have to use guards before accessing them... but you're maintaining the existing level of safety which is fine for this pr